app/services/prison_visits/api.rb

```diff
@@ -59,15 +59,19 @@ def get_slots(
       prison_id:, prisoner_number:, prisoner_dob:, use_nomis_slots: false
     )
       response = @client.get(
-        '/available_slots',
+        '/slots',
         params: {
-          prison_id: prison_id,
-          prisoner_number: prisoner_number,
-          prisoner_dob: prisoner_dob,
-          use_nomis_slots: use_nomis_slots
+          prison_id: prison_id, prisoner_number: prisoner_number,
+          prisoner_dob: prisoner_dob, use_nomis_slots: use_nomis_slots,
+          start_date: Time.zone.today, end_date: 28.days.from_now.to_date
```

Comment:
I had the same thought. I've kept it part of the API call interface as it allows for more flexibility and the overhead seemed a fairly low price to pay.
I will change it a date range though, still not used to use this awesome standard that should be implemented in ruby...